xml/en/docs/http/ngx_http_upstream_module.xml

```diff
@@ -948,7 +948,8 @@ in the specified <value>zone</value>.
 
 <para>
 <note>
-This directive is available as part of our
+Starting from version 1.17.5 up to version 1.27.3,
+this directive was available as part of our
```

Comment:
This reads as it is no longer available.
I suggest to rewrite similar to syslog:

This directive is available since version 1.17.5 as part of our
<commercial_version>commercial subscription</commercial_version>.

(Other documentation such as the "max_conns" parameter could be improved in the same way.)

Reply:
please consider the following version - it's unified with the already existing variant + added "only" that gives some hint:

Since version 1.17.5 and prior to version 1.27.3,
this directive was available only as part of our
<commercial_version>commercial subscription</commercial_version>.

Reply:
The variant "Since version X and prior to version Y" duplicates Y in `<appeared-in>`, so I'd refrain from such wording.  Still, we need to specify X for commercial subscription, hence my proposal.

Adding "only" barely clarifies IMHO, because we already have all necessary information in `<appeared-in>`, it just can make reading worse. 

And to repeat, "was" doesn't look correct to me, because the feature remains available in relevant versions.